packages/markdown/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@prairielearn/markdown",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "main": "./dist/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PrairieLearn/PrairieLearn.git",
+    "directory": "packages/markdown"
+  },
+  "scripts": {
+    "build": "tsc",
```

Comment:
@nwalters512 it doesn't need to be in this PR, but I'd imagine there would need to be a different build (esbuild? compiled-assets?) for this to work in client side, right?

Reply:
That's correct; we'd be bundling, not just transpiling.